Placeholder in itempreview added

jmferreirab opened the pull request:
# Description

Conditional rendering for item.image when it comes empty (aka "")

jmferreirab committed:
Placeholder in itempreview added

vanessa-cooper commented:
Thanks jmferreirab, I'll review the changes right away!

vanessa-cooper reviewed (changes requested):
Make sure all checks pass, only then you can merge the PR.

jmferreirab committed:
add or operator

jmferreirab committed:
run prettier

To run prettier, had to run sudo apt-get update, sudo apt-get install npm, npx prettier --write .

vanessa-cooper reviewed (approved):
Excellent job jmferreirab! You're now able to merge the PR.
